app/main.py

```diff
@@ -2,0 +1,27 @@
+import datetime
+from typing import List, Dict, Any
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: List[Dict[str, Any]], cafe: Cafe) -> str:
+    for friend in friends:
+        if "vaccine" not in friend:
+            return "All friends should be vaccinated"
+
+    not_vaccinated_count = sum(
+        1 for friend in friends
+        if friend["vaccine"]["expiration_date"] < datetime.date.today())
+    mask_needed_count = sum(
+        1 for friend in friends
+        if not friend.get("wearing_a_mask", True))
+
+    if not_vaccinated_count > 0:
+        return "All friends should be vaccinated"
```

Comment:
Code Efficiency issue: You are returning the same string `All friends should be vaccinated` in two different places (lines 9 and 19). You can refactor this by creating a variable for this string and use it in both places.